app/controllers/chat_channel_memberships_controller.rb

```diff
@@ -3,6 +3,7 @@ class ChatChannelMembershipsController < ApplicationController
 
   def find_by_chat_channel_id
     @membership = ChatChannelMembership.where(chat_channel_id: params[:chat_channel_id], user_id: current_user.id).first
+    not_found unless @membership
```

Comment:
Can I suggest replacing this with:

```ruby
ChatChannelMembership.where(...).first!
```

?

Rails has many handy ActiveRecord methods to bail out in case items don't exist, see https://api.rubyonrails.org/classes/ActiveRecord/FinderMethods.html#method-i-first-21

Reply:
Sure! `not_found` was suggested in the issue description and I just went with it. It's replaced now :)